manifests/0000_50_olm_06-psm-operator.deployment.ibm-cloud-managed.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     include.release.openshift.io/ibm-cloud-managed: "true"
 spec:
   strategy:
-    type: RollingUpdate
+    type: Recreate
```

Comment:
@tmshort this will slow down the new pod from coming up, but will prevent the new pod from attempting to acquire a lease before the other pod has shut down, meaning that it won't wait the retry duration, which should yield overall speed improvements.